code/modules/mob/living/simple_animal/abnormality/zayin/bottle.dm

```diff
@@ -69,8 +68,10 @@
 			new /obj/item/ego_weapon/eyeball(get_turf(user))
```

Comment:
Is it intentional to be able to spawn the weapon without dying? I'd move this to line 73 instead if not. Iirc the weapon obtained here can scale to be quiet powerful.